camayoc/tests/qpc/cli/csv_report_parsing.py

```diff
@@ -0,0 +1,135 @@
+# coding=utf-8
+"""Helper functions for parsing csv reports."""
+
+import csv
+
+EXPECTED_SUMMARY_REPORT_ID_FIELDS = [
```

Comment:
Maybe it is better to make this a tuple to avoid changing and give a more constant meaning?

Reply:
Thanks, I had this before but removed it trying to solve an unrelated issue. It also fits the structure of the expected lists in other files. I'll switch back to this.